db/schema.rb

```diff
@@ -40,15 +40,15 @@
     t.integer "likes_counter"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
-    t.bigint "author_id", null: false
+    t.bigint "author_id"
     t.index ["author_id"], name: "index_posts_on_author_id"
   end
 
   create_table "users", force: :cascade do |t|
     t.string "name"
     t.string "photo"
     t.string "bio"
-    t.string "posts_counter"
+    t.integer "posts_counter"
```

Comment:
Out of your 4 tests, 3 were failing, but after fixing the issues above only one failed. Implementing the suggestion below should fix it also.

- [ ] Your tests one of your tests fail because this `posts_counter` should be an `integer` type, not `string`.

  3 out of 4 failing | 1 out of 4 failing
  :---:|:---:
  ![eue](https://user-images.githubusercontent.com/81468675/200134943-9de54926-e3eb-405e-9e7a-5964684e4280.jpg) | ![yrur](https://user-images.githubusercontent.com/81468675/200134945-279b35eb-40d3-400d-afe4-f2cde88ce48a.jpg)